sdk/formrecognizer/ai-form-recognizer/review/ai-form-recognizer.api.md

```diff
@@ -271,10 +299,12 @@ export interface FormText {
 // @public
 export class FormTrainingClient {
     constructor(endpointUrl: string, credential: TokenCredential | KeyCredential, options?: FormRecognizerClientOptions);
+    beginCopyModel(modelId: string, target: CopyAuthorization, options?: BeginCopyModelOptions): Promise<PollerLike<PollOperationState<CustomFormModelInfo>, CustomFormModelInfo>>;
     beginTraining(trainingFilesUrl: string, useTrainingLabels?: boolean, options?: BeginTrainingOptions<FormModelResponse>): Promise<PollerLike<PollOperationState<FormModelResponse>, FormModelResponse>>;
     deleteModel(modelId: string, options?: DeleteModelOptions): Promise<RestResponse>;
     readonly endpointUrl: string;
     getAccountProperties(options?: GetAccountPropertiesOptions): Promise<AccountProperties>;
+    getCopyAuthorization(resourceId: string, resourceRegion: string, options?: GetCopyAuthorizationOptions): Promise<CopyAuthorization>;
```

Comment:
Correct me if I didn't see it, but we want to support the ability to pass the CopyAuthorization details to another user or straight into the REST API. For python, we don't return the strongly-typed model and instead just return a dict for this reason. In .NET, I believe they expose a convenience method `toJson` which lets the user accomplish this. @maririos

Reply:
If you are talking about converting the object to json string,  this is mostly done in JS using `JSON.stringify(obj)`

Reply:
With renaming the properties will this still have the desired output? aka could I pass this to REST API?

Reply:
No that won't work, so the api we want here is to return a string that is compatible with the REST service?  I could add a method but I think a better name is needed here to avoid confusion vs normal to json string.

Reply:
On the other hand, JS object can be used like a dictionary too

Reply:
Compatible with REST service or other language SDKs. I shouldn't have to do a lot of renaming to the copy authorization you give me if I want to pass it into my python code.

Reply:
I will merge the original response object with this convenience object (similar to what Python does I guess)

Reply:
See how in .NET I parse it so it will work with any language or REST API like Krista said https://github.com/Azure/azure-sdk-for-net/blob/master/sdk/formrecognizer/Azure.AI.FormRecognizer/src/CopyAuthorization.cs#L42

Reply:
Done. I updated it to be more like Python. I have `expiresOn` commented out because it is not working for Production env and the fix is available in GA according to Chris.  What's the purpose of having this field?  Mainly user friendly?